org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/SbtBuildReporter.scala

```diff
@@ -87,7 +87,7 @@ private[zinc] class SbtBuildReporter(project: IScalaProject) extends xsbti.Repor
       val markerPos = MarkerFactory.RegionPosition(offset, identifierLength(pos.lineContent, pos.pointer), line)
       BuildProblemMarker.create(resource, severity, msg, markerPos)
     } else
-      logger.info("suppressed error in Java file: %s".format(msg))
+      logger.info(s"suppressed error in Java file ${resource.getFullPath}:$line: $msg")
```

Comment:
Oh, I misread your comment. I thought you asked why we don't create a marker. Yes, it makes sense. I'll change the error level.